src/System.Text.RegularExpressions/src/System/Text/RegularExpressions/RegexCharClass.cs

```diff
@@ -47,18 +47,18 @@ internal sealed class RegexCharClass
         private const char ZeroWidthJoiner = '\u200D';
         private const char ZeroWidthNonJoiner = '\u200C';
 
-        private static readonly string s_internalRegexIgnoreCase = "__InternalRegexIgnoreCase__";
-        private static readonly string s_space = "\x64";
-        private static readonly string s_notSpace = "\uFF9C";
-        private static readonly string s_word = "\u0000\u0002\u0004\u0005\u0003\u0001\u0006\u0009\u0013\u0000";
-        private static readonly string s_notWord = "\u0000\uFFFE\uFFFC\uFFFB\uFFFD\uFFFF\uFFFA\uFFF7\uFFED\u0000";
-
-        public static readonly string SpaceClass = "\u0000\u0000\u0001\u0064";
-        public static readonly string NotSpaceClass = "\u0001\u0000\u0001\u0064";
-        public static readonly string WordClass = "\u0000\u0000\u000A\u0000\u0002\u0004\u0005\u0003\u0001\u0006\u0009\u0013\u0000";
-        public static readonly string NotWordClass = "\u0001\u0000\u000A\u0000\u0002\u0004\u0005\u0003\u0001\u0006\u0009\u0013\u0000";
-        public static readonly string DigitClass = "\u0000\u0000\u0001\u0009";
-        public static readonly string NotDigitClass = "\u0000\u0000\u0001\uFFF7";
+        private const string InternalRegexIgnoreCase = "__InternalRegexIgnoreCase__";
+        private const string Space = "\x64";
+        private const string NotSpace = "\uFF9C";
+        private const string Word = "\u0000\u0002\u0004\u0005\u0003\u0001\u0006\u0009\u0013\u0000";
+        private const string NotWord = "\u0000\uFFFE\uFFFC\uFFFB\uFFFD\uFFFF\uFFFA\uFFF7\uFFED\u0000";
+
+        public const string SpaceClass = "\u0000\u0000\u0001\u0064";
+        public const string NotSpaceClass = "\u0001\u0000\u0001\u0064";
+        public const string WordClass = "\u0000\u0000\u000A\u0000\u0002\u0004\u0005\u0003\u0001\u0006\u0009\u0013\u0000";
+        public const string NotWordClass = "\u0001\u0000\u000A\u0000\u0002\u0004\u0005\u0003\u0001\u0006\u0009\u0013\u0000";
+        public const string DigitClass = "\u0000\u0000\u0001\u0009";
+        public const string NotDigitClass = "\u0000\u0000\u0001\uFFF7";
```

Comment:
Another "please make this `internal const` so we don't have to look at the type definition to realize public was a mistake"

Reply:
Sure